docs/vi/index.md

```diff
@@ -0,0 +1,326 @@
+---
+layout: defaut
+title: Học sâu
+author: Alfredo Canziani
+lang-ref: home
+lang: vi
+translation-date: 16 Sept 2020
+translator: Huynh Nguyen
+---
+
+**DS-GA 1008 · Học kỳ mùa xuân 2020 · [Trung tâm khoa học dữ liệu, NYU](http://cds.nyu.edu/)**
+
+| Người hướng dẫn | Yann LeCun & Alfredo Canziani |
+| Bài giảng    | Thứ hai 16:55 – 18:35, [GCASL C95](http://library.nyu.edu/services/campus-media/classrooms/gcasl-c95/) |
+| Thực hành    | Thứ ba 19:10 – 20:00, [GCASL C95](http://library.nyu.edu/services/campus-media/classrooms/gcasl-c95/) |
+| [PIAZZA](https://piazza.com/nyu/spring2020/dsga1008/home)      | Mã truy cập: `DLSP20` |
+| Tài liệu    | [Google Drive](https://bitly.com/DLSP20), [Notebooks](https://github.com/Atcold/pytorch-Deep-Learning) |
+    
+## Mô tả khóa học
+
+Khóa học này liên quan đến các kỹ thuật mới nhất trong học sâu (Deep learning) và học biểu diễn (Representation learning), nội dung tập trung vào học sâu có giám sát (Supervised earning) và không giám sát (Unsupervised learning), phương pháp nhúng (Embedding method), học số liệu (Metric learning), mạng tích chập (Convolutional) và lặp lại (recurrent nets), với các ứng dụng cho thị giác máy tính (Computer vision), hiểu ngôn ngữ tự nhiên (Natural language understanding) và nhận diện giọng nói (Speech recognition). Các điều kiện tiên quyết, bao gồm: Giới thiệu về khoa học dữ liệu ([DS-GA 1001](https://cds.nyu.edu/academics/ms-curriculum/)) hoặc các khóa học máy học cấp độ sau đại học.
+
+## Bài giảng
+
+**Chú giải**: 🖥 Slides, 📓 Jupyter notebook, 🎥 YouTube video.
+
+<table>
+<!-- =============================== HEADER ================================ -->
+  <thead>
+    <tr>
+      <th>Tuần</th>
+      <th align="left">Định dạng</th>
+      <th align="left">Tiêu đề</th>
+      <th align="left">Tài liệu tham khảo</th>
+    </tr>
+  </thead>
+  <tbody>
+<!-- =============================== WEEK 1 ================================ -->
+    <tr>
+      <td rowspan="3" align="center"><a href="en/week01/01">①</a></td>
+      <td rowspan="2">Bài giảng</td>
+      <td><a href="week01/01-1">History and motivation</a></td>
```

Comment:
Same here and elsewhere.
Also, why is this written in English?

Reply:
I will translate the titles later.